src/Components/Footer/Footer.tsx

```diff
@@ -1,14 +1,25 @@
+import styles from "./Footer.module.css";
+
 export const Footer = () => {
   return (
-    <div>
-      <div>
+    <section className={styles.footerSection}>
+      <div className={styles.footer}>
+        {/* <div>
         <ul>
           <li>The Royal Canine</li>
           <li>business hours</li>
           <li>location</li>
         </ul>
+      </div> */}
+        <div className={styles.brandName}>
```

Comment:
Let's center this all the way. 
![image](https://github.com/Kenzie-Tech-Solutions/theRoyalCanine/assets/88916004/8666e757-6f92-4d45-96b8-8f5a908651cc)